### ISSUE TITLE
standalone SPM

### ISSUE THREAD

**Issue by bilgelm** [TimVanMourik/GiraffeTools#113]
**Is your enhancement suggestion related to a problem? Please describe.**
Giraffe is using SPM via Matlab, which requires a licence.

**Describe the solution you'd like**
Add support for standalone SPM (and make this the default).

**Comment by TimVanMourik** [comment]
I'd happy happy to support this! Do you know about current solutions that implement running MATLAB in Docker? Is this implemented in Neurodocker, @kaczmarj? That would up the neurodocker integration in the priority list.

**Comment by bilgelm** [comment]
Yes, Neurodocker supports standalone SPM. Matlab is not necessary for this - only the Matlab runtime compiler.

**Comment by kaczmarj** [comment]
@bilgelm is correct. though i have had issues with certain docker base images. `(neuro)debian:stretch` definitely works, so i would recommend basing images on that.

**Comment by TimVanMourik** [comment]
A temporary solution has been implemented by grabbing a standard SPM template. 

Neurodocker implementation to be concluded (see https://github.com/TimVanMourik/NeurodockerWeb), but at the moment there's a working solution in place. 

Example in:
https://giraffe.tools/porcupine/TimVanMourik/GiraffePlayground/b51beaee18262d80fc4a1b268165d7aacf85df77
